FoodOrderingSystem.py

```diff
@@ -0,0 +1,925 @@
+                else:
+                    print("Items not removed")
+
+            elif option == CUST_OPT_UPDATE_FOOD_TO_ORDER:
+                food_id = input("Enter food ID: ")
+                food_qty = input("Enter food quantity: ")
+                update = self.customer.update_food_to_order(o.order_id, food_id, food_qty, session)
+                if update:
+                    print("Items updated")
+                else:
+                    print("Items not updated")
+
+            option = int(input(selection))
+        
+        if o:
+            print("\nYour order is generated. \nOrder number: {}".format(o.order_id))
+        else:
+            print("Order not generated. Please try again!")
+
+    def view_order(self, order_id):
+        session = self.Session()
+        view_order_item = self.customer.view_order_per_item(order_id, session)
+        if view_order_item:
+            for fc, fd, cos in view_order_item:
+                print("\nFood category: {} \nFood name: {} \nFood price: {} \nFood quantity: {} \nTotal per item: {}".format(fc.name, fd.food_name, fd.price, cos.food_qty, (fd.price*cos.food_qty)))
+
+        view_order_grand = self.customer.view_order_grand_total(order_id, session)
+        if view_order_grand:
+            for fd, cos, cd, cosa, t in view_order_grand:
+                print("\nCustomer name: {} \nOrder ID: {} \nTotal bill: {}".format(cd.cust_name, cosa.order_id, t))
+
+    def checkout(self, order_id, order_status, order_address, checkout_time, estimated_time, bill_amount):
+        session = self.Session()
+        c = self.customer.checkout(order_id, order_status, order_address, checkout_time, estimated_time, bill_amount, session) 
+        if c:
+            print("Checkout sucessful!")
+        else:
+            print("Checkout not sucessful!") 
+
+    def cancel_order(self, order_id, order_status):
+        session = self.Session()
+        c = self.customer.cancel_order(order_id, order_status, session) 
+        if c:
+            print("Cancel sucessful!")
+        else:
+            print("Cancel not sucessful!") 
+        
+    def view_orders_status(self, order_id):
+        session = self.Session()
+        view_order_item = self.customer.view_order_per_item(order_id, session)
+        if view_order_item:
+            for fc, fd, cos in view_order_item:
+                print("\nFood category: {} \nFood name: {} \nFood price: {} \nFood quantity: {} \nTotal per item: {}".format(fc.name, fd.food_name, fd.price, cos.food_qty, (fd.price*cos.food_qty)))
+                        
+        view_status = self.customer.view_orders_status(order_id, session)
+        if view_status:
+            for cos, cd, cosa, dp in view_status:                        
+                print("\nCustomer name: {} \nOrder ID: {} \nDeliver person name: {} \nOrder status: {} \nTotal price: {}".format(cd.cust_name, cosa.order_id, dp.delivery_person_name, cosa.order_status, cosa.bill_amount))
```

Comment:
This line is too long! Lines should never be too long in python, i.e. it should not cross 79th column. You can do this to make it short:

```python
print(
    "\nOrderID: {} \nDeliverer: {} \nStatus: {} \nPrice: {}".format(
        cd.cust_name, cosa.order_id,
        dp.delivery_person_name, cosa.order_status, cosa.bill_amount
    )
)
```

Do it for all the lines that exceed 79 characters!